server/requirements-dev.txt

```diff
@@ -8,3 +8,4 @@ pytest-mock
 flake8
 flake8-bugbear
 black
+lxml-stubs
```

Comment:
where is this needed?

Reply:
This is not really needed, but it is really useful in development for getting linting and auto-completion in the IDE with `lxml`. They are just external type annotations since the `lxml` is mostly compiled code and can not be directly used by the static type linter. I found it [here](https://github.com/microsoft/pylance-release/issues/392)

Reply:
Oh, this is nifty thanks for sharing.